.erb-lint.yml

```diff
@@ -7,34 +7,18 @@ linters:
     exclude:
       - '*/app/views/account_reset/*'
       - '*/app/views/accounts/*'
-      - '*/app/views/devise/*'
-      - '*/app/views/idv/address/*'
-      - '*/app/views/idv/come_back_later/*'
       - '*/app/views/idv/confirmations/*'
       - '*/app/views/idv/capture_doc/*'
       - '*/app/views/idv/doc_auth/*'
-      - '*/app/views/idv/forgot_password/*'
-      - '*/app/views/idv/gpo/*'
-      - '*/app/views/idv/otp_delivery_method/*'
-      - '*/app/views/idv/otp_verification/*'
       - '*/app/views/idv/phone/*'
-      - '*/app/views/idv/review/*'
       - '*/app/views/idv/shared/_document_capture.html.erb'
-      - '*/app/views/forgot_password/*'
       - '*/app/views/event_disavowal/*'
-      - '*/app/views/events/*'
-      - '*/app/views/layouts/*'
       - '*/app/views/mfa_confirmation/*'
-      - '*/app/views/pages/*'
-      - '*/app/views/partials/*'
-      - '*/app/views/password_capture/*'
       - '*/app/views/reactivate_account/*'
       - '*/app/views/saml_idp/*'
```

Comment:
I don't know why it was here in the first place, and not related to your changes, but I think we can remove this one as well while we're here.

```suggestion
```